test-loop-tests/src/tests/continuous_epoch_sync.rs

```diff
@@ -0,0 +1,143 @@
+use near_async::time::Duration;
+use near_epoch_manager::epoch_sync::derive_epoch_sync_proof_from_last_final_block;
+use near_o11y::testonly::init_test_logger;
+use near_primitives::hash::CryptoHash;
+use near_primitives::version::{PROTOCOL_VERSION, ProtocolFeature};
+use near_store::adapter::StoreAdapter;
+use near_store::adapter::epoch_store::EpochStoreAdapter;
+
+use crate::setup::builder::TestLoopBuilder;
+use crate::utils::account::{create_validators_spec, validators_spec_clients};
+use crate::utils::node::TestLoopNode;
+
+// Test that epoch sync proof is correctly updated after each epoch.
+// Validate the updated proof against derive_epoch_sync_proof_from_last_final_block.
+#[test]
+fn test_epoch_sync_proof_update() {
```

Comment:
Confirmed that both tests pass with `ProtocolFeature::ContinuousEpochSync` enabled